src/models/Author.js

```diff
@@ -0,0 +1,25 @@
+import { DataTypes } from "sequelize";
+import { sequelize } from "../utils/database.js";
+
+const Author = sequelize.define('Author', {
+    id: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        primaryKey: true,
+        allowNull: false
+    },
+    name: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    biography: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    bornDate: {
+        type: DataTypes.DATE,
+        allowNull: false
+    }
```

Comment:
(optional) Check the way in which we can add timestamps too.

Reply:
yes added this using 
`{
        timestamps: true,
    }`